app/views/catalog/_home_text.html.erb

```diff
@@ -26,6 +26,1171 @@ to your own application: <br />
 <div id="about">
   <h3 class='section-heading'><a href="/rails/info/properties">About your application&rsquo;s environment</a></h3>
   <div id="about-content" class="well" style="display: none"></div>
+  <div id="about-content2" class="well" style="display: none">
+
+  <!--div class="container"-->
+
+  <!-- Navbar
+  ================================================== -->
```

Comment:
Thanks. I talked to Linda some and some of this process is a little clearer. I'll try again.